packages/Liquid-Tests.package/LQPollTests.class/instance/setUp.st

```diff
@@ -0,0 +1,4 @@
+running
+setUp
+	super setUp.
+	PollRepo default: LocalObjectRepo new.
```

Comment:
[sand, not-scope-of-this-pr] we need to do this quite a lot, right? maybe there's some space for abstraction here, where we could extract `PollRepo default:` or `LiquidParticipantMenu answeredPollIds:  Set new` into one / multiple common test classes.

Reply:
Correct, I also had some problems with `PollRepo default` and finding out when we need to "reset" it. Would you mind to explain your idea further? What do you mean by abstracting the behaviour? We cannot just define a new class `LiquidTestPollRepo` and use `LiquidTestPollRepo default`, since the call to `PollRepo default` is not done in our tests, but within e.g. `LiquidPoll>>startWithId: ifTaken:`.

Reply:
We could have an abstract `LiquidTestCase` class that resets the PollRepo as part of its setup, which our actual tests could extend from

Reply:
I see. This could indeed solve some problems. Since is a "bigger" change, I would like to get @PaulVII's opinion, I personally would approve

Reply:
Maybe we could have a LQTestUtils class or similar with setupLocalRepo and similar methods? But I'd also be fine with keeping it as it is

Reply:
What do you mean by "keeping it as it is"? Setting PollRepo default in setUp of each test that neds it? And if we add a superclass `LiquidTestCase` I would just implement setUp, there which just sets PollRepo default to a new LocalObjectRepo. Tests which need a RemoteServer will need to do that themselves anyways

Reply:
> I added an abstract superclass in the newest commit, please both @PaulVII and @Skn0tt take a look if you like it, else we can remove it again to work on it further

I put the comment in the wrong conversation :|